### PR TITLE
To-Do List For Review Purposes

### PR DESCRIPTION
**Pull request by HENRYKC24**
### In this pull request, I:
1. Created a new empty repo.
2. Copied my code from the "To Do list" project and commit it as one big chunk of code in a feature branch.
3. Opened a pull request.
4. Asked my partner for a review in GitHub.
5. Shared the pull request link with my partner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+[Live Demo Link](https://henrykc24.github.io/to-do-list-draggable-review/dist/)
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+Clone the repo with `git clone https://github.com/HENRYKC24/to-do-draggable-review.git`
```
Comment (elmar8287) [inline README.md]: Dear Henry, here is a link to clone is not as a repo name, it should be "to-do-list-dragable-review" with ..list..

File: src/manipulateToDo.js [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+          });
+        }
+        node.style.backgroundColor = '#4c3c3c';
+        node.style.color = '#fff';
+      });
+      setTimeout(() => {
+        // document.getElementById(id).style.backgroundColor = "#fff";
+        let elUnder = document.getElementById(id);
+        // document.getElementById(id).style.backgroundColor = "#4c3c3c";
+        const classArray = elUnder.classList.value.split(' ');
```
Comment (elmar8287) [inline src/manipulateToDo.js]: You have some comments in the code. It will be more nice and more professional to remove them.

**Review by elmar8287 — COMMENTED** [review]
## Hi @HENRYKC24 👋🏾👋🏾👋🏾 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
#### Highlights

- items render dynamically on page load ✔️ 
- I can drag items easily ✔️ 
- good GitHub workflow ✔️ 
- not forming any linter errors ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

File: README.md (hunk 1 above)
Comment (HENRYKC24) [reply README.md]: Thank you so much @elmar8287 for your effort in making my project better.
Could you please explain more on the above point:pray:

File: src/manipulateToDo.js (hunk 2 above)
Comment (HENRYKC24) [reply src/manipulateToDo.js]: Oh! my bad. I will work on all these, my friend. Thanks

File: README.md (hunk 1 above)
Comment (HENRYKC24) [reply README.md]: Thank you for clarifying this in the zoom call. I am really grateful:pray:

**Comment by HENRYKC24** [comment]
### Thanks for your great job so far in reviewing my project:+1:
#### Now I have tried to fix all the errors in my code and I want to ask for your re-review, please:pray:
#### You may also want to check other things I fixed from my personal review and morning session review [here](https://github.com/HENRYKC24/to-do-list-draggable/issues)
### Happy Coding:tada:

**Comment by HENRYKC24** [comment]
Hi Elmar:raised_hand:
I want to bring it to your notice that I have fixed the grey out issue on dragging and dropping a task:smile:
You can check that out while re-reviewing my code:pray:
Thank you.

**Comment by elmar8287** [comment]
### Congratulations @HENRYKC24 ! 💯 🚀 

Great job! your project works very well 👏🏾 Excellent work!

It is time to merge it.

Happy codding!🎉🎉🎉

**Comment by HENRYKC24** [comment]
## Thank you so much Elmar:pray:

**Pull request merged**